source/dotnet/Library/AdaptiveCards/AdaptiveAction.cs

```diff
@@ -49,5 +49,15 @@ public abstract class AdaptiveAction : AdaptiveTypedElement
 #endif
         [DefaultValue("default")]
         public string Style { get; set; } = "default";
+
+        /// <summary>
+        ///    When set false, action is disabled 
+        /// </summary>
+        [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
```

Comment:
What's the difference between `DefaultValueHandling.Ignore` and `DefaultValueHandling.IgnoreAndPopulate`?

Reply:
https://www.newtonsoft.com/json/help/html/t_newtonsoft_json_defaultvaluehandling.htm